src/runners/scilla_runner.ml

```diff
@@ -132,21 +162,8 @@ let deploy_library (cli : Cli.ioFiles) gas_remaining =
 
       (* Checking initialized libraries! *)
       let gas_remaining' = check_libs clibs elibs cli.input gas_remaining in
+      let _ = validate_get_init_json cli.input_init gas_remaining' lmod.smver in
```

Comment:
```suggestion
      let () = validate_get_init_json cli.input_init gas_remaining' lmod.smver in
```

Reply:
Oh,right. The return type of `validate_get_init_json` is not unit :) I should have used a typechecker, sorry for that